core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -726,6 +729,15 @@ public static String getOncoKBApiUrl()
         return "";
     }
 
+    public static String getCivicUrl() {
+        String civicUrl = properties.getProperty(CIVIC_URL);
```

Comment:
I would .trim() this. Unfortunately a white space at the end of the property makes the call fail.